spec/classes/datadog_agent_spec.rb

```diff
@@ -593,6 +596,20 @@
                 'content' => /^sd_jmx_enable: true\n/,
                 )}
             end
+            context 'with extra_template enabled' do
+                let(:params) {
+                    {:extra_template => 'custom_datadog/extra_template_test.erb',
```

Comment:
I think I added this fixture correctly, I know ignore this directory in `.gitignore`, but it seemed like the correct thing to do? Please advise!